tests/suites/os/tests/connectivity/index.js

```diff
@@ -133,10 +147,25 @@ module.exports = {
 						await this.context
 							.get()
 							.worker.executeCommandInHostOS(
-								`ping -c 10 ${URL_TEST}`,
+								`curl ${URL_TEST}`,
 								this.context.get().link,
 							);
-						test.true(`${URL_TEST} responded over ${proxy} proxy`);
+
+						const proxyLog = await this.context
+							.get()
+							.worker.executeCommandInHostOS(
+								['balena',
+									'logs',
+									await getProxyContainerID(),
+									'|', 'tail', '-n1'].join(' '),
+								this.context.get().link,
+							);
+
+						test.match(
+							proxyLog,
+							/\[(socks5|http)\] 127\.0\.0\.1:[0-9]* <->/,
```

Comment:
```suggestion
							/\[${proxy}\] 127\.0\.0\.1:[0-9]* <->/,
```
Make sure we aren't matching logs from the last test by restricting proxy type. Need to convert `http-connect` to `http` before this step for a match to be found?

Reply:
Also does this string not require quotes to pass as a parameter?

Reply:
@klutchell I had done this earlier, setting a `protocol` variable and matching against that instead of this pattern. I deleted it for simplicity, but I agree that it's better to explicitly verify that we're getting requests over the right protocol.

> Also does this string not require quotes to pass as a parameter?

Right, it's weird, but JavaScript apparently allows defining regular expressions that aren't really strings. This was news to me too.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions